docs/app-starter.stories.mdx

```diff
@@ -53,17 +53,19 @@ The order of these CSS stylesheets is **very important**:
 
 No matter how you choose to import the following, this order must be upheld.
 
+**We recommend you use your `tailwind.css` file to ensure this order (JS files will order them differently)**
+
 ### Reset
 
 Kaizen's CSS reset aims to add any global default styles such as typography or base styles which enable Tailwind utilities (you can think of this as Kaizen's own Preflight).
 
 Note: this is different from a Normalize which aims to create uniformity in basic elements across the multiple browsers. (eg. Corrects bugs and common browser inconsistencies).
```

Comment:
Can we also explain in a sentence or two:
1. Why we aren't able/choosing to use Tailwind's Preflight.
2. What kinds of things our reset does and why. (Perhaps, as Tailwind's docs do, we could link to the source code and ensure it contains clear comments to document the purpose of each style rule.)

Reply:
I don't think this is the appropriate place to explain these things, and doesn't help the reader achieve the intended goal of the page, which is to let someone know how to manually retro-fit an existing code base, which should become rarer as more repos are spun up with `next-template`. 

I'll remove the reference to Preflight to reduce any potential questions, and I'd be happy to write a decision log with these details, in case someone would like to know what led us down this path.

Reply:
Sounds good. Having the reasoning for a custom preflight/reset recorded somewhere easy to find (IMO a comment in the file itself that compiles away would be ideal for this) will be valuable.